src/edu/csus/ecs/pc2/exports/ccs/ResultsFile.java

```diff
@@ -33,7 +33,13 @@ public class ResultsFile {
 
     private static final String SILVER = "Silver Medal";
 
-    private static final String HONORABLE = "Honorable" ;
+    private static final String HIGHEST_HONOR = "Highest Honors";
+
+    private static final String HIGH_HONOR = "High Honors";
+
+    private static final String HONOR = "Honors";
+
+    private static final String HONORABLE = "Honorable";
```

Comment:
I seems like the text string assigned to this enum element should be "Honorable Mention", not (just) "Honorable".  All the other enum elements related to honors have the full name of their category as their value; this one is inconsisent in that sense.  (I don't know if it would at some point make a difference -- but I can imagine that it _could_.)

Reply:
The results file has always used Honorable.  It may be a 'CMS' thing...

[https://github.com/icpcsysops/wfluxor/blob/main/contests%2Fwf46%2Ffinals%2Fresults%2Fprimary%2Fresults.tsv](https://github.com/icpcsysops/wfluxor/blob/main/contests%2Fwf46%2Ffinals%2Fresults%2Fprimary%2Fresults.tsv)

Reply:
I agree; let's not change this at least for now.